### PR TITLE
PLT-7130 adding more generic push notificaiton message.

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: webapp/components/admin_console/push_settings.jsx [hunk 1]
```diff
@@ -207,6 +207,7 @@ export default class PushSettings extends AdminSettings {
                 <DropdownSetting
                     id='pushNotificationContents'
                     values={[
+                        {value: 'generic_no_channel', text: Utils.localizeMessage('admin.email.genericNoChannelPushNotification', '"Send generic description with only sender name')},
                         {value: 'generic', text: Utils.localizeMessage('admin.email.genericPushNotification', 'Send generic description with user and channel names')},
```
Comment (hmhealey) [inline webapp/components/admin_console/push_settings.jsx]: This should be updated to "sender and channel names"

File: webapp/i18n/en.json [hunk 2]
```diff
@@ -272,6 +272,7 @@
   "admin.email.enableEmailBatchingDesc": "When true, users can have email notifications for multiple direct messages and mentions combined into a single email, configurable in <b>Account Settings > Notifications</b>.",
   "admin.email.enableEmailBatchingTitle": "Enable Email Batching:",
   "admin.email.fullPushNotification": "Send full message snippet",
+  "admin.email.genericNoChannelPushNotification": "Send generic description with only sender name",
   "admin.email.genericPushNotification": "Send generic description with user and channel names",
```
Comment (hmhealey) [inline webapp/i18n/en.json]: Same here

File: webapp/components/admin_console/push_settings.jsx (hunk 1 above)
Comment (coreyhulen) [inline webapp/components/admin_console/push_settings.jsx]: Sorry, I don't follow?

generic_no_channel = sender name
generic (was the pre-existing one) = sender and channel names

Comment (coreyhulen) [inline webapp/components/admin_console/push_settings.jsx]: I didn't want to change the existing one "generic".

Comment (coreyhulen) [inline webapp/components/admin_console/push_settings.jsx]: ^ @hmhealey FYI

Comment (hmhealey) [inline webapp/components/admin_console/push_settings.jsx]: @coreyhulen The new help text doesn't match the option in the dropdown. It has "Send generic description with sender and channel names", but the dropdown option uses "user and channel names"

**Comment by coreyhulen** [comment]
@hmhealey strings should be updated.

**Pull request merged**